cli/requirements.txt

```diff
@@ -4,4 +4,5 @@ wheel>=0.24.0, <1.0
 pytest>=2.9.1
 teamcity-messages>=1.20
 PyInstaller==3.2.1
+retrying==1.3.3
```

Comment:
For the record, this requirements.txt file seems ignored so we had to add the dependency in setup.py, I'll create a ticket about this.

Reply:
see https://jira.mesosphere.com/browse/DCOS_OSS-1762?filter=-1